Gopkg.toml

```diff
@@ -0,0 +1,111 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
```

Comment:
Its going to merge tomorrow but I think should this be aligned with k8s 1.16.

https://github.com/giantswarm/chart-operator/blob/86b2a7abdf27c2449fdb0f67f576ea6368cdc1a4/Gopkg.toml

cc @sslavic who did the hard stuff on this

Reply:
It's 1.16 now